spa/src/components/place/home/main2d.vue

```diff
@@ -135,9 +136,30 @@ export default Vue.extend({
         console.log(error);
       }
     },
+    async checkAdmin() {
+      try {
+        await this.$http.get(
+          `/block/${  this.$store.data.place.block.id  }/can_admin`,
+        );
+        this.canAdmin = true;
+      } catch (e) {
+        console.log(e);
+      }
+    },
   },
   mounted() {
     this.getData();
+    this.checkAdmin();
+  },
+  watch: {
```

Comment:
A watch maybe needed but is the better thing to watch for might be this.$route.params.username